GVRf/Framework/framework/src/main/java/org/gearvrf/animation/GVRAnimation.java

```diff
@@ -393,6 +439,12 @@ final boolean onDrawFrame(float frameTime) {
         return stillRunning;
     }
 
+    public void animate(float timeInSec)
```

Comment:
Seems like it doesn't have to be public. Please remove the qualifier or add javadoc. Preferably remove the qualifier.